entries/QUnit.module.xml

```diff
@@ -15,16 +15,34 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="nested" optional="true">
+			<desc>A callback with grouped tests and nested modules to run under the current module label.</desc>
+			<type name="Function">
+				<argument name="hooks" type="Object"></argument>
+			</type>
+		</argument>
 	</signature>
 	<desc>
 		Group related tests under a single label.
 	</desc>
 	<longdesc>
 		<p>
-			All tests that occur after a call to <code>QUnit.module()</code> will be grouped into that module, until other <code>QUnit.module()</code> is called. The test names will all be preceded by the module name in the test results. You can then use that module name to select tests to run.
+			You can then use the module name to organize, select and filter tests to run.
+		</p>
+		<p>
+			All tests inside a module callback function will be grouped into that module. The test names will all be preceded by the module name in the test results. Other modules can be nested inside this callback function, where their tests names will be labeled by their names recursively prefixed by their parent modules.
+		</p>
+		<p>
+			If <code>QUnit.module</code> is defined without a <code>nested</code> callback argument, all subsequently defined tests will be grouped into the module until another module is defined.
+		</p>
+		<p>
+			Modules with grouped tests functions allow defining nested modules, and QUnit will run tests on the parent module before going deep on the nested ones, even if they're declared first. <code>beforeEach</code> and <code>afterEach</code> callbacks on a nested module call will stack to the parent hooks.
```

Comment:
How does the stacking first? FIFO? FILO? Maybe use "queuing" instead of "stacking"?